android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/RibConnector.kt

```diff
@@ -0,0 +1,52 @@
+package com.badoo.ribs.core.routing
+
+import android.os.Bundle
+import com.badoo.ribs.core.Node
+
+interface RibConnector {
+    /**
+     * Implies attaching child node + view
+     */
+    fun attachChild(childRouter: Node<*>, bundle: Bundle? = null)
```

Comment:
parameter naming is clashing with actual value.
childRouter -> but it is not a router but Node.

The same for other methods in this interface.

Reply:
Good point, should be renamed to childNode.